draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-map-items-reg-procedures title="CoMID Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.language | RFC-AAAA
+| 1 | comid.tag-identity | RFC-AAAA
+| 2 | comid.entity | RFC-AAAA
+| 3 | comid.linked-tags | RFC-AAAA
+| 4 | comid.triples | RFC-AAAA
+| 5-255 | Unassigned
+{: #tbl-iana-comid-map-items title="CoMID Map Items Initial Registrations"}
+
+{: #iana-comid-xxx-map-items}
+## CoMID Tag-Identity-Map Items Registry
+
+This document defines a new registry titled "CoMID Tag-Identity Map".
+The registry uses integer values as index values for items in 'tag-identity-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-tag-identity-map-items-reg-procedures title="CoMID Tag-Identity Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Tag-Identity Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.tag-id | RFC-AAAA
+| 1 | comid.tag-version | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-comid-tag-identity-map-items title="CoMID Tag-Identity Map Items Initial Registrations"}
+
+{: #iana-comid-entity-map-items}
+## CoMID Entity-Map Items Registry
+
+This document defines a new registry titled "CoMID Entity Map".
+The registry uses integer values as index values for items in 'comid-entity-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-entity-map-items-reg-procedures title="CoMID Entity Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Entity Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.entity-name | RFC-AAAA
+| 1 | comid.reg-id | RFC-AAAA
+| 2 | comid.role | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-comid-entity-map-items title="CoMID Entity Map Items Initial Registrations"}
+
+{: #iana-comid-linked-tag-map-items}
+## CoMID Linked-Tag-Map Items Registry
+
+This document defines a new registry titled "CoMID Linked-Tag Map".
```

Comment:
linked-tag-map is not extensible today and should remain so until a use case is defined that motivates. It is however extensible by $tag-id-type-choice (which aligns with tag-identity-map). These should remain consistently defined.

Reply:
I have a slightly different take on this.  It doesn't seem implausible -- well, it's actually quite natural -- that one might want to add [target attributes](https://datatracker.ietf.org/doc/html/rfc8288#section-2.2) to this.  In fact, I think we should add it upfront and lock the type as Ned suggests.